src/shim/config/config.toml

```
@@ -0,0 +1,17 @@
+# Only support rune right now.
+runtime_class = "rune"
+
+log_level = "info" # "trace" "debug" "info" "warn" "error" "fatal" "panic"
+
+[containerd]
+    # Pre-installed OCI bundle for agent enclave container
+    agent_container_instance = "/opt/enclave-cc/agent-instance/"
+    # The root dir of agent enclave container running state
+    agent_container_root_dir = "/run/containerd/agent-enclave/"
+    # The listening address of the agent container
+    # Note that the default URL will listen on all network interfaces.
+    # TODO: After occlum releases NGO at the end of 2022, the URL will
+    # switch to the Unix domain socket.
+    agent_url = "tcp://0.0.0.0:7788"
```

we want localhost don't we
```suggestion
    agent_url = "tcp://127.0.0.1:7788"
```